gulpfile.js

```diff
@@ -1,27 +0,0 @@
-var gulp = require('gulp'),
```

Comment:
I don't really use these and I'm not a fan of gulp. I guess for now just run ```npm start```, ```npm run build``` and ```npm test```, and if you want to watch... ```nodemon``` for running and ```watch "npm test" src``` for tests